### PR TITLE
54 frontend that loads placeholder loginregistration

### PR DESCRIPTION
**Pull request by BarryFullStack**
login and registration page is placeholder functional, no endpoints

### CODE REVIEW

**Comment by BarryFullStack** [comment]
merge conflicts have been corrected.

File: FE/src/components/Login.jsx [hunk 1]
```diff
@@ -1,8 +1,176 @@
-import { Link } from 'react-router-dom'
+import {useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
 
-export default function Login ({token}) {
+export default function Login ({token, setToken}) {
+    const [email, setEmail] = useState('');
+    //const [password, setPassword] = useState('');
```
Comment (ljreaux) [inline FE/src/components/Login.jsx:6]: remove

File: FE/src/components/Login.jsx [hunk 2]
```diff
@@ -1,8 +1,176 @@
...
+    const [loginPassword, setLoginPassword] = useState('');
+    const [registerPassword, setRegisterPassword] = useState('');
+    //const [username, setUsername] = useState('');
```
Comment (ljreaux) [inline FE/src/components/Login.jsx:9]: remove

File: FE/src/components/Login.jsx [hunk 3]
```diff
@@ -1,8 +1,176 @@
...
+                },
+                body: JSON.stringify({
+                    username: loginUsername,
+                    password: loginPassword,
+                })
+            });
+            const userAuth = await response.json();
+            if (userAuth.token) {
+                setToken(userAuth.token);
+                alert(userAuth.message); //do we have a message for this for successful login?
```
Comment (ljreaux) [inline FE/src/components/Login.jsx:35]: we will, yes

File: FE/src/components/Login.jsx [hunk 4]
```diff
@@ -1,8 +1,176 @@
...
+                navigate('/account');
+            } else {
+                setFailMessage(userAuth.message); //do we have a message for a failed login?
```
Comment (ljreaux) [inline FE/src/components/Login.jsx:38]: We will

File: FE/src/components/Login.jsx [hunk 5]
```diff
@@ -1,8 +1,176 @@
...
+            }
+        } catch (error) {
+            setError(error.message); //do we have an error message?
```
Comment (ljreaux) [inline FE/src/components/Login.jsx:41]: I think so, I will look.

**Review by ljreaux — APPROVED** [review]

**Review by jared23m — APPROVED** [review]

**Review by amead99 — APPROVED** [review]

**Pull request merged**